frontend/src/metabase/collections/components/CollectionSidebar/Bookmarks/Bookmarks.tsx

```diff
@@ -51,6 +54,12 @@ const CollectionSidebarBookmarks = ({
   bookmarks,
   deleteBookmark,
 }: CollectionSidebarBookmarksProps) => {
+  const storedShouldDisplayBookmarks =
```

Comment:
I believe it's a user-level setting and it should be set on the backed. @alxnddr @noahmoss Could you please guide us with defining a new user-level setting?

Reply:
@ranquild Ultimately I agree. 

Just working iteratively, we can add this later and add value incrementally.

If it's 3 lines we can copy and paste, sure, let's just do it now.

Reply:
@ranquild @gusaiani User-level settings can be defined easily on the backend now, but I realized that we don't have APIs that will allow regular users to easily read or write them. Our current setting endpoints are restricted to admin users.

I'll need to make a some changes to enable non-admins to use user-local settings. If this is blocking you I can prioritize it today or tomorrow.

Reply:
@gusaiani would it be ok to wait for a day until there is a proper way to save user settings? would it block you?

Reply:
@ranquild @noahmoss 

I propose we merge this is as is and add a follow-up PR integrating the back-end once it's ready.

Waiting wouldn't block me, strictly speaking.

But if we merge it as is I could wait even 5 workdays and not have a hanging PR, reducing cognition overload in the process.

Reply:
Most importantly people could get a taste of it in stats right away.

Reply:
OK, I'll review this PR shortly

Reply:
@gusaiani here you go: https://github.com/metabase/metabase/pull/21235

This allows user-local settings to be fetched and written by all users via `GET /api/setting/:key`, `PUT /api/setting` and `PUT /api/setting/:key`. The only thing still restricted to admins is listing all settings via `GET /api/setting`.

I'll try to get this reviewed in the next day or two.

Reply:
@noahmoss Thank you very much, Noah.

We'll integrate soon.